docs/troubleshoot/ze/ze-known-limits.md

```diff
@@ -12,7 +12,7 @@ With Remote SSH, the network protocol provides users with a secure connection to
 
 Both WSL and Remote SSH provide more tools for mainframe developers, but they also have limitations when it comes to credentials.
 
-Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, not the Linux environment or the remote server or virtual machine. This is because credential storage is managed by VS Code, which stores them using the host's operating system credential manager.
+Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, API Mediation Layer tokens &mdash; resides on the developer’s local machine, provided the developer is not connected to a remote environment through VS Code. If you are connected to a remote environment through VS Code, your secure credentials are stored on the remote system.
```

Comment:
```suggestion
Authentication information used in Zowe Explorer &mdash; such as usernames and passwords, SSH keys, and API Mediation Layer tokens &mdash; reside on the developer’s local machine, provided the developer is not connected to a remote environment through VS Code. If you are connected to a remote environment through VS Code, your secure credentials are stored on the remote system.
```

Reply:
@traeok -- technically, "information" is singular, therefore "resides" is correct.